patches/components-content_settings-core-browser-host_content_settings_map.cc.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/components/content_settings/core/browser/host_content_settings_map.cc b/components/content_settings/core/browser/host_content_settings_map.cc
+index 5d066ff91e41f83e6b3cff8b76c2021f65e690c4..d6649bcd727c01107bc89f1eee7008d5c9384d2e 100644
+--- a/components/content_settings/core/browser/host_content_settings_map.cc
++++ b/components/content_settings/core/browser/host_content_settings_map.cc
+@@ -305,6 +305,8 @@ HostContentSettingsMap::HostContentSettingsMap(PrefService* prefs,
+   content_settings_providers_[ProviderType::kDefaultProvider] =
+       std::move(default_provider);
+ 
++  BRAVE_CREATE_REMOTE_LIST_PROVIDER
```

Comment:
i wonder if we can manually add the provider with `RegisterProvider` ?

Reply:
I looked pretty hard but didn't find a way to do that consistently whenever a HostContentSettingsMap is instantiated.

Reply:
but isn't it always created with a factory? i.e. we could chromium_src `HostContentSettingsMapFactory::BuildServiceInstanceFor`

Reply:
Good point. I have done it that way now.